libs/openFrameworks/gl/ofGLUtils.cpp

```diff
@@ -655,6 +659,7 @@ int ofGetNumChannelsFromGLFormat(int glFormat){
 	switch(glFormat){
 	case GL_RGB:
 		return 3;
+	case GL_BGRA:
```

Comment:
i think you are missing an ifdef here otherwise this fails in other opengles platforms

Reply:
Done